scripts/eol/scanner.py

```
@@ -0,0 +1,74 @@
+
+    try:
+        data = json.loads(pkg.read_text())
+        engines = data.get("engines", {})
+        node = engines.get("node")
+
+        if not node:
+            return None
+
+        m = re.search(r"(\d+)", node)
+        return f"{m.group(1)}.0" if m else None
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)